### PR TITLE
updates javascript intro challenges

### PR DESCRIPTION
**Pull request by elysemontano**
updates javascript intro challenges to show code examples and outputs.

### CODE REVIEW

File: javascript/intro.md [hunk 1]
```diff
@@ -316,44 +316,98 @@ console.log(greeting.slice(1, 3))
...
+console.log(theAnswer / 2)  // => 21
+
+// Write the code that will log the length of theQuestion.
+console.log(theQuestion.length)  // => 34
+
+// Write the code that will log the index of the character "f" in the theQuestion.
+console.log(theQuestion.indexOf("f"))  // => 2
+
+// Write the code that will log the concatenation of the two variables.
+console.log("The answer to " + theQuestion + " is " + theAnswer)  // => The answer to life, the universe, and everything is 42
```
Comment (sjproctor) [inline javascript/intro.md:398]: Pop this "The answer to life, the universe, and everything is 42" and this "the universe" into strings.

**Review by sjproctor — CHANGES_REQUESTED** [review]
Just a small fix!

File: javascript/intro.md (hunk 1 above)
Comment (elysemontano) [reply javascript/intro.md:398]: Not sure I follow...  "life, the universe, and everything" are already strings in a variable and was being asked to concatenate the variables.  I added the strings "the answer to " and " is " to make it a full sentence.  Did you want these set as variables too?

**Review by sjproctor — APPROVED** [review]
Fabulous!

**Pull request merged**